File: src/test/java/model/moves/BaseMoveTest.java
package model.moves;

import model.util.Position;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class BaseMoveTest {

    @Test
    void equals() {
        Assertions.assertEquals(
                new BaseMove(new Position(0, 0), new Position(0, 1)),
                new BaseMove(new Position(0, 0), new Position(0, 1))
        );
    }
}

File: engine/src/test/java/engine/moves/BaseMoveTest.kt
package engine.moves

import engine.util.Position
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class BaseMoveTest {

    @Test
    fun equals() {
        Assertions.assertEquals(
                BaseMove(Position(0, 0), Position(0, 1)),
                BaseMove(Position(0, 0), Position(0, 1))
        )
    }
}